pydicom/tests/test_pillow_pixel_data.py

```diff
@@ -287,11 +287,6 @@ def teardown(self):
         """Reset the pixel data handlers."""
         pydicom.config.pixel_data_handlers = self.original_handlers
 
-    def test_raises_if_endianess_not_set(self):
```

Comment:
No longer needed because `utils.pixel_dtype()` gets the endianness from the transfer syntax UID.